lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -706,12 +726,21 @@ protected function getAlterColumnCommentSQL($tableName, $columnName, $comment)
      */
     protected function getDropColumnCommentSQL($tableName, $columnName)
     {
+        if (strpos($tableName, '.') !== false) {
+            list($schema, $table) = explode('.', $tableName);
+            $schema               = $this->quoteStringLiteral($schema);
```

Comment:
It's better to name the variables as `$tableSQL` and `$schemaSQL` since they are not names but SQL expressions.